math/binomial.cpp

```diff
@@ -0,0 +1,73 @@
+/**
+ * @file
+ * @brief Program to calculate binomial coefficients.
+ *
+ * @author astron
```

Comment:
you can provide link to your github profile, if you wish.
Doxygen supports Markdown, you can use markdown like
`[Name](link to profile)`